Gemfile

```diff
@@ -2,8 +2,10 @@ source "https://rubygems.org"
 
 gem 'middleman', "~> 3.3.0"
 gem 'zurb-foundation', '~> 4.0'
+gem 'bourbon', '~> 4.0.2'
```

Comment:
fwi, I recently forked it and after middleman started up and I wanted to look at the website, I got this:

![error](http://puu.sh/gM4OB/0f9c2e0af3.png)

I bumped the version of bourbon to 4.2.1 and it works fine now. Guess it has to do with having set a default value in https://github.com/thoughtbot/bourbon/commit/25f235be3cbba5a65a4c37ce3127c76c7182e5c0

I will submit the version bump as PR together with the template I am about to add.